src/modules/search/components/helpers.js

```diff
@@ -94,9 +94,16 @@ export const indexFiles = async client => {
   const notOrphans = file =>
     folders.find(folder => folder._id === file.dir_id) !== undefined
   const notRoot = file => file._id !== ROOT_DIR_ID
+  // Shared drives folder to be hidden in search.
+  // The files inside it though must appear. Thus only the file with the folder ID is filtered out.
+  const notSharedDrivesDir = file => file.id !== SHARED_DRIVES_DIR_ID
```

Comment:
You should test on `_id` rather than `id`.  `id` might work thanks to cozy-client support, but we better not rely on this much

Reply:
yes good catch that was my initial intent